src/apiserver/api_server_impl.cc

```diff
@@ -428,7 +428,22 @@ void APIServerImpl::RegisterGetTable() {
                           writer << err.Set("Invalid path");
                           return;
                       }
+                      std::vector<std::string> dbs;
+                      hybridse::sdk::Status status;
+                      auto ok = sql_router_->ShowDB(&dbs, &status);
+                      if (!ok) {
+                          writer << err.Set(status.msg);
+                          return;
+                      }
                       auto db = db_it->second;
+                      bool db_ok = std::find(dbs.begin(), dbs.end(), db) != dbs.end();
+                      if (!db_ok) {
+                          writer.StartObject();
+                          writer.Member("code") & -1;
+                          writer.Member("msg") & std::string("DB not found");
+                          writer.EndObject();
```

Comment:
You could use `GeneralError` to simplify the code. No need to start/end object. Please ref other methods, e.g. `RegisterGetSP()`.